cms/djangoapps/contentstore/management/commands/backfill_orgs_and_org_courses.py

```diff
@@ -62,11 +62,11 @@ def handle(self, *args, **options):
         """
         Handle the backfill command.
         """
-        orgslug_coursekey_pairs = find_orgslug_coursekey_pairs()
-        orgslug_library_pairs = find_orgslug_library_pairs()
+        orgslug_courseid_pairs = find_orgslug_courseid_pairs()
+        orgslug_libraryid_pairs = find_orgslug_libraryid_pairs()
```

Comment:
Most of the diff in the file is changing renaming things from "key" to "id". There is an informal convention in edx-platform that a "key" is an OpaqueKey instance, whereas an "id" is a stringified key.